text/3487-feature-visibility.md

```diff
@@ -0,0 +1,165 @@
+- Feature Name: feature-metadata
+- Start Date: 2023-09-08
+- RFC PR: [rust-lang/rfcs#3487](https://github.com/rust-lang/rfcs/pull/3487)
+- Rust Issue:
+  [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+
+[summary]: #summary
+
+This RFC describes a new key under `features` in `Cargo.toml` to indicate that a
+feature is private.
+
+Please see the parent meta RFC for background information: [`feature-metadata`].
+
+# Motivation
+
+[motivation]: #motivation
+
+Cargo feature gates are one of the main means to support optional build
```

Comment:
nitpick: We usually call it just "Cargo features". Also, conditional builds and optional dependencies sound more complete to me

```suggestion
[Cargo features](link to features doc) are one of the main means to support conditional build and optional dependency
```